verkle-trie/Cargo.toml

```diff
@@ -15,7 +15,7 @@ ark-ff = { version = "^0.3.0", default-features = false }
 ark-ec = { version = "^0.3.0", default-features = false }
 ark-serialize = { version = "^0.3.0", default-features = false }
 verkle-db = { path = "../verkle-db", default-features = false }
-
+tokio = { version = "1.21.1", features = ["full"] }
```

Comment:
why is this needed?

Reply:
It isn't will remove. I was toying around with async so it slipped in